python/pyspark/sql/tests/test_dataframe.py

```diff
@@ -742,6 +742,79 @@ def test_df_show(self):
             message_parameters={"arg_name": "truncate", "arg_type": "str"},
         )
 
+    def test_df_merge_into(self):
```

Comment:
This should actually be skipped if we build without test. let me make a quick followup

Reply:
https://github.com/apache/spark/pull/48404